app/api/predictions/service.py

```diff
@@ -89,73 +90,44 @@ async def get_new_prediction(cls, prompt: str, client_websocket):
             except Exception as e:
                 print(f"Unable to connect to Inference Server. Retrying in {os.environ.get('RETRY_TIME')} seconds:", e)
                 retry_counts += 1
-                await asyncio.sleep(int(os.environ.get('RETRY_TIME'))) 
+                await asyncio.sleep(int(os.environ.get('RETRY_TIME')))
```

Comment:
What if `RETRY_TIME` is not defined? You probably want a fall-back value.

Reply:
For line 67 (from a prev PR): 
`ws://{os.environ.get('AKASH_ENDPOINT')}"`
I would parameterized the protocol (`ws`) with a fall-back value.